File: app/src/main/java/com/seniorzhai/dagger2sample/view/imp/IMeiziView.java
package com.seniorzhai.dagger2sample.view.imp;

import android.widget.ImageView;

/**
 * Created by zhai on 16/5/23.
 */

public interface IMeiziView {

    ImageView getImageView();

    void onPrepareSave();

    void onSaveSuccess();

    void onSaveFail();

    void onSaveFinish();
}

File: app/src/main/java/com/seniorzhai/dagger2sample/view/imp/IMeiziView.kt
package com.seniorzhai.dagger2sample.view.imp

/**
 * Created by zhai on 16/5/23.
 */

interface IMeiziView {

    fun loadImage(url: String)

    fun onPrepareSave()

    fun onSaveSuccess()

    fun onSaveFail()

    fun onSaveFinish()
}
